CHANGELOG.md

```diff
@@ -13,4 +18,4 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 -   Minor layouting stuff
 
 [Unreleased / SNAPSHOT]: https://github.com/ie3-institute/TUDoBeamerTheme/compare/v1.0...HEAD
-[v1.0]: https://github.com/ie3-institute/TUDoBeamerTheme/releases/tag/v1.0
\ No newline at end of file
+[v1.0]: https://github.com/ie3-institute/TUDoBeamerTheme/releases/tag/v1.0
```

Comment:
FMI: Why doesn't git notice that nothing changend here? Is that a recurrent issue?

Reply:
I think it' just a displaying issue. More or less by accident, I added a new line at the end of the document. So, that has changed. But the blank line isn't shown here...